### DISCUSSION TITLE
ScriptReference/SceneView-lastActiveSceneViewChanged

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#519] [Notes]
# ScriptReference/SceneView-lastActiveSceneViewChanged

https://docs.unity3d.com/ScriptReference/SceneView-lastActiveSceneViewChanged.html

**Comment by MechWarrior99** [comment]
The first parameter is the previously active SceneView, the second parameter is the newly active SceneView.
